data/map planets.txt

```diff
@@ -91,12 +93,12 @@ planet Columbia
 		fleet "Large Militia" 14
 
 planet "San Paolo"
-	attributes border factory farming tourism
+	attributes periphery factory farming tourism
 	landscape land/ncity6
-	description "San Paolo is a tropical world, dominated by vast jungles that stretch across continents, in the plains and valleys where the trees don't dominate the landscape exist many settlements and farms that reach millions of kilometers, all to feed the ever increasing population of the urban centers, where the skyline is dominated by skyscrapers who house millions working in some variation of goods and services or manufacturing. San Paolo is also home to Taurus-Ikaros Defense Group, one of the largest weapon manufacturers in the south."
-	description "As the first planet to be settled in this region, its customs and culture differ somewhat from the other planets in the Periphery but its still recognizable. Many people from all corners of space come here for the pristine beaches here during the summer. "
+	description "San Paolo is a tropical world, dominated by vast jungles that stretch across continents, in the plains and valleys where the trees don't dominate the landscape exist many settlements and farms that reach millions of kilometers, all to feed the ever increasing population of the urban centers, where the skyline is dominated by skyscrapers who house millions working in some variation of goods and services or manufacturing. San Paolo is also home to Taurus-Ikaros Defense Group, one of the largest weapon manufacturers in the South."
+	description "	As the first planet to be settled in this region, its customs and culture differ somewhat from the other planets in the Periphery but its still recognizable. Many people from all corners of space come here for the pristine beaches here during the summer."
```

Comment:
San Paolo like Austral has a significant criminal underground, unlike Austral though the mafias here operate a variety of business as fronts. Construction is the most significant. A mayor corruption scandal resulted in the system loosing influence to Paradeigm.